src/index.ts

```diff
@@ -1,7 +1,11 @@
 export {
     PartialConfig as AwsRumConfig,
-    Orchestration as AwsRum
+    Orchestration as AwsRum,
+    PartialCookieAttributes as PartialCookieAttributes,
+    PageIdFormat as PageIdFormat,
+    Telemetry as Telemetry
 } from './orchestration/Orchestration';
```

Comment:
can really even be 

```ts
export * from from './orchestration/Orchestration';
```

if we are not changing anything that is exported

Reply:
Unfortunately we are using alias for some exports :P